inlang/source-code/editor/src/pages/@host/@owner/@repository/State.tsx

```diff
@@ -208,9 +208,6 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 	// get lix errors
 	const [lixErrors, setLixErrors] = createSignal<ReturnType<Repository["errors"]>>([])
 	createEffect(() => {
-		// reset errors to empty on repo changes
-		setLixErrors([])
-
 		repo()?.errors.subscribe((errors) => {
 			setLixErrors(errors)
 		})
```

Comment:
@NiklasBuchfink why can't you directly subscropbe to `repo().errors`? 

Does no wrapper for repo reactivity exist?

Reply:
You are right, moved it behind the repo resource and removed the effect and reset. Everything works, including the resets after changing the branch 👍